### DISCUSSION TITLE
TIMEOUT

### DISCUSSION THREAD

**Discussion by IsidoraGuinez** [IIC2133-PUC/2024-2#193] [Ayuda]
Hola ,
Quería preguntar sobre el error de "timeout" que me salió en algunos test de la T2.No entiendo muy bien por qué pasa , ya que al correrlo en mi compu no me sale este error.
Cuales podrían ser las causas de este error?
Gracias de antemano :)

**Comment by tomastrivino** [comment]
Ese error significa que tu testcase se ejecutó en un tiempo superior al límite. Si el límite es de 5 segundos y tu test se ejecutó en 10 segundos, entonces es un TIMEOUT, y por lo tanto tiene 0 puntos.

Para ver cuándo demora tu código, utiliza el comando `time`.

Saludos!

**Comment by IsidoraGuinez** [comment]
aaah y puede ser que en la corrección se demore más ? 
Es que lo probe en mi compu y se demora como 3s en real

**Reply by tomastrivino** [reply]
Perdón por la demora.
Entiendo que para cada tarea se entrega una imagen de Docker para probar. De todos modos,  si la diferencia es muy drástica puedes recorregir.
